desktop/src/cg/group4/sensor/DesktopAccelerationStatus.java

```diff
@@ -1,12 +1,16 @@
 package cg.group4.sensor;
 
 
+import cg.group4.util.sensors.AccelerationState;
+import cg.group4.util.sensors.AccelerationStatus;
+import cg.group4.util.subscribe.Subject;
+
 /**
  * Desktop implementation of thee acceleration status.
  */
-public class DesktopAccelerationStatus extends AccelerationStatus {
-
+public class DesktopAccelerationStatus implements AccelerationStatus {
 
+    protected Subject uselessSubject = new Subject();
```

Comment:
The app returns an NullPointerException if you don't instantiate that variable to a new Subject object. This happens because when you instantiate a stroll, you want to also add an observer to the AccelerationStatus subject.